docs/code-examples/line_segments2d_simple_v2.py

```diff
@@ -0,0 +1,14 @@
+"""Log a simple set of line segments."""
+import numpy as np
+import rerun as rr
+import rerun.experimental as rr2
+
+rr.init("line_segments2d", spawn=True)
+
+rr2.log(
+    "segments",
+    rr2.LineStrips2D(np.array([[0, 0], [2, 1], [4, -1], [6, 0]]).reshape([2, 2, 2])),
+)
```

Comment:
I find the usage of reshape here to be a lot more confusing than being explicit.
```suggestion
rr2.log(
    "segments",
    rr2.LineStrips2D(
        np.array(
            [
                [[0, 0], [2, 1]],
                [[4, -1], [6, 0]],
            ]
        )
    ),
)
```